starknet_devnet/starknet_wrapper.py

```diff
@@ -135,13 +143,11 @@ async def initialize(self):
             self.__initialized = True
 
     async def create_empty_block(self):
```

Comment:
we can add -> StarknetBlock: here

Reply:
Addressed in https://github.com/Shard-Labs/starknet-devnet/pull/389/commits/9185ea1284fc5f75b2d21b2c2bcb1083c19eb1c2